llvm/cmake/config-ix.cmake

```diff
@@ -27,6 +27,12 @@ if (CMAKE_SYSTEM_NAME STREQUAL "NetBSD")
   set(NETBSD 1)
 endif()
 
+# Confirmed in
+# https://github.com/llvm/llvm-project/pull/104706#issuecomment-2297153534
+if (CMAKE_SYSTEM_NAME MATCHES "OS390")
+  set(ZOS 1)
```

Comment:
The port of cmake to z/OS, in `Modules/Platform/OS390.cmake`, already has this line in it.  Will this be a problem?

```
set(OS390 1)
set(ZOS 1)
```